Tests/SwiftParserTest/translated/TypeExprTests.swift

```diff
@@ -550,27 +579,117 @@ final class TypeExprTests: XCTestCase {
     )
   }
 
-  func testTypeExpr26() {
+  func testCompositionTypeExpr() {
     AssertParse(
       """
-      protocol P1 {}
-      protocol P2 {}
-      protocol P3 {}
-      func compositionType() {
-        _ = P1 & P2 
-        _ = P1 & P2.self 
-        _ = (P1 & P2).self // Ok.
-        _ = (P1 & (P2)).self // FIXME: OK? while `typealias P = P1 & (P2)` is rejected.
-        _ = (P1 & (P2, P3)).self 
-        _ = (P1 & Int).self 
-        _ = (P1? & P2).self 
-        _ = (P1 & P2.Type).self 
-        _ = P1 & P2 -> P3
-        _ = P1 & P2 -> P3 & P1 -> Int
-        _ = (() -> P1 & P2).self // Ok
-        _ = (P1 & P2 -> P3 & P2).self 
-        _ = ((P1 & P2) -> (P3 & P2) -> P1 & Any).self // Ok
-      }
+      _ = P & Q
```

Comment:
Were all the test cases above moved/rewritten to some other file? I didn’t do a full search but I can’t find the equivalent of `(P1 & (P2)).self` straight away.

Reply:
These are from the updates to the translated file in apple/swift#62775